### ISSUE TITLE
ios_command calling "reload cancel" fails

### ISSUE THREAD

**Issue by ninnux** [ansible-collections/cisco.ios#272]
Hi, I'm working on some ansible playbook and I noted that
calling "reload cancel" by ios_commnad sometime it fails for timeout.
It fails 20% of times.
I seams it happen because reload cancel do not give a clear console: it waits for the shell until command_timeout occurs.
Could you help me?
Thank you
Nino Ciurleo

**Comment by justjais** [comment]
@ninnux Thanks for logging the issue, wanted to recheck if you're still seeing the respective issue as you've mentioned that it occurs 20% of the time and rest of the time it works as expected

**Comment by justjais** [comment]
@ninnux the issue is inactive and is waiting on the info we’re closing the issue, plz reopen a new issue if you’re still facing the reported issue

**Comment by Vebryn** [comment]
Same here

**Comment by Vebryn** [comment]
workaround using cli_command with sendonly parameter and a sleep after reload cancel :

```
- cli_command:
    command: reload cancel
    sendonly: yes

- pause:
    seconds: 1
```

issue seems due to shutdown aborted message :

```
cisc#reload cancel
cisc#


***
*** --- SHUTDOWN ABORTED ---
***

cisc#
```

message comes during next task. Task doesn't recognize cisco prompt.
